### DISCUSSION TITLE
is possible to use functionary with ollama ?

### DISCUSSION THREAD

**Discussion by grigio** [MeetKai/functionary#127] [General]
also ollama has openai function callin support, but it seems the tools api isn't supported

**Comment by sarahlmk** [comment]
Hi sorry for the late response! Currently we don't support Ollama as almost all our models use prompt templates where the tools descriptions are presented in Typescript schema and tool calls are not in JSON format. Ollama currently does not support tool models where the tool calls are not in json format.
